src/infrastructure/sqlalchemy/model.py

```
@@ -1,18 +1,16 @@
-from sqlalchemy import Column, String, Integer, DateTime, CHAR, Text, Date, func
+from sqlalchemy import Column, String, Integer, DateTime, CHAR, Text, Date, func, UniqueConstraint
```

Ruff (0.8.2)

`sqlalchemy.UniqueConstraint` imported but unused

Remove unused import: `sqlalchemy.UniqueConstraint`

(F401)